cfgov/ask_cfpb/tests/models/test_pages.py

```diff
@@ -72,6 +73,10 @@ def setUp(self):
             SubCategory, name='stub_subcat', parent=self.category, _quantity=3)
         self.category.subcategories.add(self.subcategories[0])
         self.category.save()
+        self.social_image = mommy.prepare(CFGOVImage)
+        self.social_image.save()
```

Comment:
You can use [`mommy.make`](http://model-mommy.readthedocs.io/en/latest/basic_usage.html#basic-usage) to prepare and save the object in one step.